myia/frontends/pytorch.py

```diff
@@ -183,3 +183,9 @@ def _to_abstract(self, v: torch.nn.Parameter, **kwargs):
         }),
         {SHAPE: tuple(v.shape), TYPE: PyTorchTensor},
     )
+
+
+__all__ = [
+    'mod_sub',
+    'pytorch_dtype_to_type',
+]
```

Comment:
I would argue that this shouldn't have an `__all__`

Reply:
I'd keep `pytorch_dtype_to_type` I think.